tests/metrics/breakdown_tests.py

```diff
@@ -0,0 +1,198 @@
+                assert 10000 < elem["samples"]["span.self_time.sum.us"]["value"] < 15000
+                assert elem["samples"]["span.self_time.count"]["value"] == 1
+                asserts += 1
+            elif elem["span"] == {"type": "db", "subtype": "mysql"}:
+                assert elem["samples"]["span.self_time.count"]["value"] == 1
+                assert 5000 < elem["samples"]["span.self_time.sum.us"]["value"] < 10000
+                assert elem["transaction"] == {"name": "test", "type": "request"}
+                asserts += 1
+    assert asserts == 3
+
+
+def test_nested_spans(elasticapm_client):
+    transaction = elasticapm_client.begin_transaction("request")
+    time.sleep(0.005)
+    with elasticapm.capture_span("test", span_type="template", span_subtype="django"):
+        time.sleep(0.005)
+        with elasticapm.capture_span("test", span_type="db", span_subtype="mysql"):
+            time.sleep(0.005)
+        with elasticapm.capture_span("test", span_type="db", span_subtype="mysql"):
+            time.sleep(0.005)
+    time.sleep(0.005)
+    elasticapm_client.end_transaction("test", "OK")
+    breakdown = elasticapm_client._metrics.get_metricset("elasticapm.metrics.sets.breakdown.BreakdownMetricSet")
+    data = list(breakdown.collect())
+    assert len(data) == 4
+    asserts = 0
+    for elem in data:
+        if "transaction.breakdown.count" in elem["samples"]:
+            assert elem["samples"]["transaction.breakdown.count"]["value"] == 1
+            assert 25000 < elem["samples"]["transaction.duration.sum.us"]["value"] < 32000
+            assert elem["transaction"] == {"name": "test", "type": "request"}
+            asserts += 1
+        elif "span.self_time.sum.us" in elem["samples"]:
+            if elem["span"] == {"type": "app", "subtype": ""}:
+                assert elem["transaction"] == {"name": "test", "type": "request"}
+                assert 10000 < elem["samples"]["span.self_time.sum.us"]["value"] < 15000
+                assert elem["samples"]["span.self_time.count"]["value"] == 1
+                asserts += 1
+            elif elem["span"] == {"type": "db", "subtype": "mysql"}:
+                assert elem["samples"]["span.self_time.count"]["value"] == 2
+                assert 10000 < elem["samples"]["span.self_time.sum.us"]["value"] < 15000
+                assert elem["transaction"] == {"name": "test", "type": "request"}
+                asserts += 1
+            elif elem["span"] == {"type": "template", "subtype": "django"}:
+                assert elem["samples"]["span.self_time.count"]["value"] == 1
+                assert 5000 < elem["samples"]["span.self_time.sum.us"]["value"] < 10000
+                assert elem["transaction"] == {"name": "test", "type": "request"}
+                asserts += 1
+    assert asserts == 4
+
+
+@pytest.mark.parametrize("elasticapm_client", [{"breakdown_metrics": False}], indirect=True)
+def test_disable_breakdowns(elasticapm_client):
```

Comment:
This test should be updated once we report `transaction.duration` even with the feature disabled.